pkg/printers/internalversion/describe.go

```diff
@@ -538,7 +539,12 @@ func describePod(pod *api.Pod, events *api.EventList) (string, error) {
 			w.Write(LEVEL_0, "Message:\t%s\n", pod.Status.Message)
 		}
 		w.Write(LEVEL_0, "IP:\t%s\n", pod.Status.PodIP)
-		w.Write(LEVEL_0, "Controllers:\t%s\n", printControllers(pod.Annotations))
+		if createdBy := printCreator(pod.Annotations); len(createdBy) > 0 {
+			w.Write(LEVEL_0, "Created By:\t%s\n", createdBy)
+		}
+		if controlledBy := printController(pod); len(controlledBy) > 0 {
+			w.Write(LEVEL_0, "Controller:\t%s\n", controlledBy)
```

Comment:
"Controller" is probably one of the most overused words in the community. How about "Controlled By"?